experimental/plugins_graphdriver.md

```diff
@@ -269,7 +269,7 @@ Get a list of changes between the filesystem layers specified by the `ID` and
 }
 ```
 
-Responds with a list of changes. The structure of a change is:
+Respond with a list of changes. The structure of a change is:
```

Comment:
But how do we deal with the sentence as follows:

**Response**:
```
{
  "Size": 512366,
  "Err": ""
}
```

Respond with the size of the new layer in bytes.
Respond with a non-empty string error if an error occurred.

Should we use "responds"?